scripts/build.sh

```diff
@@ -73,6 +73,7 @@ mkdir -p $OUTPUT/maven/org/opensearch
 cp -r ./build/local-staging-repo/org/opensearch/. $OUTPUT/maven/org/opensearch
```

Comment:
Hey @downsrob you should be adding `cp -r ./build/local-staging-repo/org/opensearch/. $OUTPUT/maven/org/opensearch` at the end after running `publishPluginZipPublicationToZipStagingRepository` task as even the zips are also copied to local `$OUTPUT` folder
Example as 
https://github.com/prudhvigodithi/job-scheduler/blob/gradleplugin/scripts/build.sh#L78-L83

Reply:
Updated, let me know if that is what you wanted. Once again I will say that it seems that you have already done this work and I feel that you could have raised this PR with far less effort.

Reply:
Hey @downsrob, this is the setting and expectation from the custom plugin end and looks good now.